shopfloor_mobile_base/static/wms/src/components/datepicker/date_picker.js

```diff
@@ -40,6 +40,7 @@ export var DatePicker = Vue.component("date-picker-input", {
                     v-model="date"
                     v-bind="attrs"
                     v-on="on"
+                    name="expiration_date"
```

Comment:
We shouldn't do this here and I would argue on why we have a specific label here if this is a generic widget.
If we really need this, we should use properties. However, I don't think you need this.

Reply:
Also, such changes to another module should have its own commit.